client/auth/emailed-login-link-successfully.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import emailValidator from 'email-validator';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import { getCurrentQueryArguments } from 'state/ui/selectors';
+
+class EmailedLoginLinkSuccessfully extends React.Component {
+	render() {
+		const { translate, emailAddress } = this.props;
+		const line = [
+			emailAddress
+				? translate( 'We sent an email to %(emailAddress)s with a magic login link.', {
+					args: {
+						emailAddress
+					}
+				} )
+				: translate( 'We sent you an email with a magic login link.' ),
+			( <br key="magic-login-line-br" /> ),
+			translate( 'It should arrive within a few minutes. Go click it!', {
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?